INSTALL.rst

```diff
@@ -564,7 +564,8 @@ When all the servers are running, it is possible to upload the demo records.
 
     $ # in a new terminal
     $ workon invenio
-    (invenio)$ inveniomanage demosite populate --packages=invenio_demosite.base
+    (invenio)$ cdvirtualenv src/invenio-demosite
+    (invenio)$ inveniomanage records create -t marcxml < invenio_demosite/testsuite/data/demo_record_marc_data.xml
```

Comment:
IMHO we can remove this part from here completely as it should be part of `invenio-records` documentation and here we should just point there.